pilot/pkg/kube/inject/mesh.go

```diff
@@ -164,6 +166,10 @@ containers:
         fieldPath: metadata.name
   - name: ISTIO_META_INTERCEPTION_MODE
     value: [[ annotation .ObjectMeta $interceptionModeKey .ProxyConfig.InterceptionMode ]]
+  [[ if (not (eq $policyCheckValue "")) -]]
```

Comment:
This is not used ( except for tests ) - the files in install/k8s... are used by sidecar injector and kubeinject, need to add this there too.